python/idsse_common/idsse/common/rabbitmq_utils.py

```diff
@@ -437,6 +437,7 @@ def __init__(
         conn_params: Conn,
         exch_params: Exch,
         *args,
+        channel: Channel | None = None,
```

Comment:
Is this the way pythonic way to specify a key/word arg with *args and **kwargs (it looks funny in between)?

Reply:
This was pylint's recommendation, although I agree it looks funny. I think it's because channel has a default value (None). 

Python doesn't like a non-required argument before `*args`, because it might cause positional args to not get picked up, but channel also can't go after `**kwargs` because the `**` swallows up any other named/keyword arguments.

Being between `*args` and `**kwargs` makes it so channel can only be passed in explicitly `channel=my_channel_var`, and won't squash any positional args that the caller wanted passed to the Thread, Publisher's parent class.